jobs/build/set_client_latest/Jenkinsfile

```diff
@@ -29,6 +29,12 @@ node {
                         $class: 'hudson.model.StringParameterDefinition',
                         defaultValue: "ocp"
                     ],
+                    [
+                        name: 'LINK_NAME',
+                        description: 'Link name to set',
```

Comment:
Can you include context in the description on what this affects? Is this closely tied to the `CHANEL_OR_RELEASE` parameter value? Any possibility of invalid input going in here causing heck on the mirror if it isn't checked for sanity first? Or, do the updates to the script account for bogus inputs fairly well?

Reply:
Nice update on the description there